src/Rule/Email.php

```diff
@@ -15,8 +15,6 @@
 use Yiisoft\Validator\SkipOnEmptyInterface;
 use Yiisoft\Validator\ValidationContext;
 
-use function function_exists;
-
```

Comment:
Can we make this without remove `use`?

Reply:
You can't mock internal function if it already exists